### ISSUE TITLE
CAs are responsible for scope of subordinate CA audits

### ISSUE THREAD

**Issue by wthayer** [mozilla/pkipolicy#182]
in https://bugzilla.mozilla.org/show_bug.cgi?id=1539296 @sleevi stated:

Ensuring that the Root CA is a responsible party for the scoping of audit engagements seems like it would be a significant positive step to improving the oversight of subordinate CAs, as well as provide a clearer sense of accountability and responsibility for that. This moves the Root CA from merely being a consumer of the audit to being an actively engaged party helping raise the quality of the audit.

**Comment by wthayer** [comment]
The root CA is ultimately responsible for ensuring that the audits for subordinate CAs they issue meet Mozilla requirements. That already seems clear in policy and also seems difficult to verify, so perhaps the idea of root CAs examining the audit engagement letters of their subordinates should become a recommended practice, i.e. https://wiki.mozilla.org/CA/Required_or_Recommended_Practices#Recommended_Practices

**Comment by BenWilson-Mozilla** [comment]
What about something like this for the Recommended Practices section?

**Audits of Third Party CAs**
The operator of a root CA is ultimately responsible for ensuring that the audits for all non-technically constrained subordinate and cross-certified CAs meet Mozilla requirements. There is a risk that such CAs, when operated externally by third parties, might not operate to the same level of care as that of the operator of the root CA. In the event that a root CA cross-signs or issues a non-technically constrained CA certificate for a CA to be operated by a third party, then the root CA should implement sufficient additional controls, such as requiring root-CA review of the auditor engagement for such third-party-operated CA to ensure a properly scoped audit. Also, because the root CA operator is responsible for updating the CCADB with such audit information when it becomes available, it should also review audit reports as soon as they are issued and before updating the information for the CA in the CCADB to ensure that the audit has covered the appropriate Mozilla criteria and that any minor non-conformities will be timely addressed. (Major non-conformities may lead to removal or distrust of the root CA.)

**Comment by sleevi** [comment]
That seems significantly weaker than what I’d originally proposed, and I’m
not sure I understand the concerns?

The gap I’m trying to close here is one where the CA says “Oh no, the Sub
CA made a mistake, but how were we supposed to know? We’re just as shocked
as you browsers, shocked I say!”

Much in the same way we’re talking about browsers being named on CAs audits
(e.g. for detailed control reporting), I was talking about ensuring that
the Root is explicitly required to be involved in the scoping of the Audit
for the Sub, in engagement and reporting. No Root can feign surprise, at
that point, because they were intimately involved. If something wasn’t
looked at, the Root can’t claim they were hoodwinked, shocked, and
flabbergasted, etc, because they were intimately involved.

To put it differently, the current requirements make it clear that the Root
is responsible for the _results_, but is otherwise permitted to feign or
choose ignorance about the process to produce those results. I’m trying to
make it clear that the Root is equally accountable for the _process_, much
as they are responsible for their own audits and auditors.

My concern with the proposed text is that it still seems to emphasize a
results-oriented approach, whereas I’m trying to make sure that the Root is
holistically responsible and unambiguously accountable, to the same degree
of their own audit.
